java/io/trino/re2j/DFA.java

```diff
@@ -60,7 +54,7 @@ private DFATooManyStatesException() {
   private final Inst[] instructions;
 
   // Search for longest match like egrep or POSIX or for first match like Perl, PCRE
-  private final MatchKind matchKind;
+  private final RE2.MatchKind matchKind;
```

Comment:
i'd suggest avoiding changes other than in `package` and `import` lines, if possible

Reply:
These were automatic too when I've renamed the package :/